07-reproj.qmd

```diff
@@ -175,9 +184,106 @@ axes[0].set_title('Original (EPSG:26912)')
 axes[1].set_title('Reprojected (EPSG:4326)');
 ```
 
-Bilinear `srtm.tif` example...
+The above example, where we automatically calculate the destination grid, is appropriate when there are no specific requirements for the raster spatial properties, but rather we just wish to preserve the raster values as much as possible (this the extent and resolution are as similar as possible to the source). 
```

Comment:
This sentence is a bit long...

Reply:
Thanks @Robinlovelace , will try to fix it now after merging